pkgs/by-name/dn/dnsmap/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, stdenv, fetchFromGitHub, autoconf, automake }:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "dnsmap";
+  version = "0.36-unstable-2024-06-16";
+
+  src = fetchFromGitHub rec {
+    owner = "resurrecting-open-source-projects";
+    repo = "dnsmap";
+    rev = "b5b4b1a7764e141f2551584d9fad3a973951eafe";
+    hash = "sha256-RTZe0kb/Fq9kIdnHQO//OP+Uop2Z5r22Z7+rQdCFsl4=";
+  };
+
```

Comment:
> strictDeps = true;

(Always have this for `stdenv.mkDerivation`. This should be the default.)

Reply:
`preConfigure`, `configureFlags` is configure phase. Should precede buildInputs here.